clients/admin-ui/src/features/datastore-connections/add-connection/sass/ConnectorParameters.tsx

```diff
@@ -117,10 +117,8 @@ export const ConnectorParameters: React.FC<ConnectorParametersProps> = ({
         });
         const payload = await createSassConnectionConfig(params).unwrap();
         dispatch(setConnection(payload.connection));
-        // Update the current browser url with the new key created
-        replaceURL(payload.connection.key, step.href);
         successAlert(`Connector successfully added!`);
-        onConnectionCreated();
+        if (onConnectionCreated) onConnectionCreated();
```

Comment:
Nit: a bit surprised we don't lint for single-line if statements, but one thing you can do here is use [optional function call](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#optional_chaining_with_function_calls): `onConnectionCreated?.()`

Reply:
I will add the curly rule to the the .eslintrc.json file.